lib/models/subscription.js

```diff
@@ -119,6 +120,17 @@ module.exports = class Subscription extends Sequelize.Model {
     return results
   }
 
+  static async agedSyncCounts () {
+    const ageThreshold = moment().subtract(7, 'd').format('YYYY-MM-DD')
+    const [ results ] = await this.sequelize.query(
+      `SELECT "syncStatus", COUNT(*) FROM "Subscriptions" WHERE "updatedAt" < :ageThreshold AND ("syncStatus" IS NULL OR "syncStatus" in ('ACTIVE', 'PENDING')) GROUP BY "syncStatus"`,
```

Comment:
I think we could save a little work in node by moving the date calculation to the sql (then we also don't need to require moment.js here)
```js
const [ results ] = await this.sequelize.query(
    `SELECT "syncStatus", 
           COUNT(*) 
    FROM   "Subscriptions" 
    WHERE  "updatedAt" < NOW - INTERVAL '7 days'
           AND ( "syncStatus" IS NULL 
                  OR "syncStatus" IN ( 'ACTIVE', 'PENDING' ) ) 
    GROUP  BY "syncStatus"`,